scripts/check_lint.sh

```diff
@@ -16,3 +16,4 @@ if [ ! $(command -v ${DOCKER_CMD}) ]; then
 fi
 
 ${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace golangci/golangci-lint:v1.21 golangci-lint run
+${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/opt/workspace -w /opt/workspace/cmd/aries-agentd golangci/golangci-lint:v1.21 golangci-lint run -c ../../.golangci.yml
```

Comment:
Consider making the linter version a variable above. We will probably forgot to update each line :).